src/wp-includes/post.php

```diff
@@ -2077,7 +2077,7 @@ function _add_post_type_submenus() {
 		if ( ! $ptype_obj->show_in_menu || true === $ptype_obj->show_in_menu ) {
 			continue;
 		}
-		add_submenu_page( $ptype_obj->show_in_menu, $ptype_obj->labels->name, $ptype_obj->labels->all_items, $ptype_obj->cap->edit_posts, "edit.php?post_type=$ptype" );
+		add_submenu_page( $ptype_obj->show_in_menu, $ptype_obj->labels->name, $ptype_obj->labels->menu_name, $ptype_obj->cap->edit_posts, "edit.php?post_type=$ptype" );
```

Comment:
Can you explain this change a little?

Reply:
This makes it so that CPTs are referred in the WP Admin sidebar menu by their `menu_name` label instead of by their `all_items` label. If there is no `menu_name` then `name` is used.

In practice it means that the sidebar will say "Navigation Menus", "Blocks", "Posts", etc. instead of "All Navigation Menus", "All Blocks", "All Posts", etc.

Re-reading the docs for `register_post_type`, though, I have this wrong. We should just set `all_items` for this CPT to "Navigation Menus" as `all_items` is specifically what is shown in the WP Admin sidebar menu.

From the docs:

> - `all_items` - Label to signify all items in a submenu link. Default is 'All Posts' / 'All Pages'.